script.domoticz.scenes/addon.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.domoticz.scenes" name="Domoticz Scenes" version="0.0.8" provider-name="Paul VW">
+  <requires>
+    <import addon="xbmc.python" version="3.0.0"/>
+     <import addon="script.module.requests"   version="1.0.0"/>
+  </requires>
+  <extension point="xbmc.python.pluginsource" library="addon.py">
+    <provides>executable</provides>
+  </extension>
+  <extension point="xbmc.addon.metadata">
+    <summary lang="nl_NL">Start scenes in Domoticz vanuit Kodi</summary>
```

Comment:
en_GB versions is mandatory (default language)

Reply:
added